prowler/providers/aws/aws_provider.py

```diff
@@ -611,16 +613,16 @@ def setup_assumed_session(
                 secret_key=assumed_role_credentials.aws_secret_access_key,
                 token=assumed_role_credentials.aws_session_token,
                 expiry_time=assumed_role_credentials.expiration,
-                refresh_using=self.refresh_credentials,
+                refresh_using=AwsProvider.refresh_credentials,
```

Comment:
I think this is not correct. `refresh_credentials` is not an static method.

Reply:
Please, @pedrooot revert this change.